mkdocs.yml

```diff
@@ -66,7 +66,7 @@ extra:
       documentation better.
 
 plugins:
-  - render_swagger
+ # - render_swagger
```

Comment:
If this is not longer needed we can remove this line entirely

Reply:
when removed rendering the docs will result in this 
![image](https://user-images.githubusercontent.com/8327751/170373839-54119e26-1ff7-47b2-a1b1-5924c9f7f39e.png)

p.s. if you want to test the docs locally you can run docker-compose up in docs directory and have it shown in localhost:8088

Reply:
Sorry - that's my bad. My swagger implementation locally is not recognized by mkdocs.  I am working on figuring that out. I forgot to uncomment this before I created the PR. I'll fix it.